apps/transactions/mixins.py

```diff
@@ -0,0 +1,18 @@
+from django.db import transaction
+from rest_framework import mixins
+from rest_framework.response import Response
+from rest_framework import status
+
+
+class CreateModelTransactionMixin(mixins.CreateModelMixin):
+    """
+    Custom Create Model Mixin with extended database transaction functionality
+    """
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+
+        if serializer.is_valid(raise_exception=True):
+            with transaction.atomic():
```

Comment:
there is no sense in atomic transaction. We don't need them foe a single operation, but good to know that you aware of this functionality